tests/test_params.py

```diff
@@ -520,3 +520,62 @@ class ParamSpider(Args[Params], Spider):
 
     with raises(ValidationError):
         get_spider(ParamSpider, kwargs={"foo": "2"})
+
+
+def test_param_hardcoding_in_subclass():
+    """When hard-coding the value of a parameter in a subclass, the default
+    value changes, but the parameter remains.
+    """
+
+    class ParentParams(BaseModel):
+        a: str = Field()
+
+    class Params(ParentParams):
+        a: str = "b"
```

Comment:
What about this, does it work the same way?

```suggestion
    class Params(ParentParams):
        a = "b"
```

Reply:
`pydantic.errors.PydanticUserError: Field 'a' defined on a base class was overridden by a non-annotated attribute. All field definitions, including overrides, require a type annotation.`

Reply:
For the record, if we wanted to override other `Field` parameters and not just the default value, it can be done with something like:

```python
    class Params(ParentParams):
        a: str = Field(**ParentParams.schema()["properties"]["a"], default="b")
```